mule-extensions-api/src/main/java/org/mule/runtime/extension/api/runtime/config/ConfigurationStats.java

```diff
@@ -35,4 +35,12 @@
   default int getRunningSources() {
     return 0;
   }
+
+  /**
+   * @return How many currently running sources are making use of the referenced configuration
+   * @since 1.2.3 1.3.0
+   */
+  default int getOpenedStreams() {
```

Comment:
Before merging this we should check that this is the api we want to use in master.

Also, there should not be a release cycle where for example, this is fixed in 4.2.x and not in 4.3.x .